### DISCUSSION TITLE
docker pull failed for OCI images with unsupported media type application/vnd.cncf.helm.config.v1+json

### DISCUSSION THREAD

**Discussion by kmeratia** [actions/actions-runner-controller#2756] [Questions]
docker pull for OCI image is failing with:
![image](https://github.com/actions/actions-runner-controller/assets/113537455/2c6571a5-7b2d-4a14-a0e8-9e915158d070)

unsupported media type application/vnd.cncf.helm.config.v1+json

docker version:
![image](https://github.com/actions/actions-runner-controller/assets/113537455/5751d15f-5ce9-4943-bc3a-d8ad7263d9f9)

**Comment by provenvelocity** [comment]
What is the fix?

**Reply by kmeratia** [reply]
use helm pull instead of docker pull

**Comment by provenvelocity** [comment]
Hum tired they also :) let me see what I did wrong Sent from my iPhoneOn Jan 8, 2024, at 5:17 PM, Krishnesh Meratia ***@***.***> wrote:﻿
use helm pull instead of docker pull
